scipy/stats/tests/test_multivariate.py

```diff
@@ -289,8 +290,8 @@ def test_pseudodet_pinv(self):
 
         # Set cond so that the lowest eigenvalue is below the cutoff
         cond = 1e-5
-        psd = _PSD(cov, cond=cond)
-        psd_pinv = _PSD(psd.pinv, cond=cond)
+        psd = _PSD(cov, atol=cond)
```

Comment:
I mean this one for example; the tolerance in these tests `atol = 1e-5` and `rtol` is not zero but `maxS*eps`. In the original test this tolerance is `1e-5 * maxS`. So needs an explicit `rtol=0` to control it precisely. 

Similarly, if this is used in other places basically this amounts to the fact that we moved the tolerances everywhere so I can imagine more things fail because of this. This was one of the reasons we stopped using the hard coded values because the tolerances were a bit too random and depended on the problems.